src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -75,12 +81,37 @@ public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentry
                 crashedLastRun = SentryNativeBridge.HandleCrashedLastRun(options);
                 PerDirectoryCrashInfo.Add(cacheDirectory, crashedLastRun);
 
-                options.DiagnosticLogger?
+                Logger?
                     .LogDebug("Native SDK reported: 'crashedLastRun': '{0}'", crashedLastRun);
             }
         }
         options.CrashedLastRun = () => crashedLastRun;
+
+        ShouldReinstallBackend = true;
     }
 
-    public static void ReinstallBackend() => SentryNativeBridge.ReinstallBackend();
+    // We're calling this in `BeforeSceneLoad` instead of `SubsystemRegistration` as it's too soon and the
+    // SignalHandler would still get overwritten.
+    [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
```

Comment:
We could have had this in here all along? :)
I dont' recall why we left on `SentryInitialization` originally, do you recall?

Reply:
The intention was to have everything annotated `RuntimeInitializeOnLoadMethod` in the `SentryInitialization.cs` so users could overwrite/remove it. But it should have belonged in `SentryNative` all along.